packages/qwik-city/middleware/request-handler/cookie.ts

```diff
@@ -52,15 +52,26 @@ const createSetCookieValue = (cookieName: string, cookieValue: string, options:
   return c.join('; ');
 };
 
+function tryDecodeUriComponent(str: string) {
+  if (str.indexOf('%') === -1) {
```

Comment:
I think this check is not-needed. If not `%`, then `decodeURIComponent` just returns a normal string.

Reply:
Yeah, agreed. I added that only because the `cookie` library had something like that (probably for optimization?).